docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
+
+![Qualifio Manager](img/qualifio_manager.png "Qualifio Manager - campaign list")
+
+## Campaign
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines at the creation. Campaign ID is a reference to the campaign in the applications, services and data connections.
+
+There are two types of campaigns:
+
+- **Anonymous** - engages with the audience, increases the advertising impressions, or collects cookies of personalized audiences.
+- **Identified** - gathers information on the user by filling out an identification form.
+
+## Publication channel
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign:
+
+- **Widget/iframe** - uses a JavaScript code or an HTML iframe from Qualifio. Additionally, you can use oEmbed, Facebook Instant articles, and AMP technologies to publish iframes. You need to manually paste the code into your website or CMS.
+
+- **Minisite** - uses a unique URL from Qualifio that can hosts the campaign. This URL points to your subdomain.
+
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
+    Each publication channel type is automatically responsive and can be used on any mobile device.
+
+## Work with campaigns
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
+To access the Qualifio Back Office page and create or manage campaigns, click the **Manage campaigns in Qualifio** button under the **Ibexa Engage** menu.
+To display your campaign on the website, use one of the available tools - Campaign block or Campaign custom tag.
+For more information about creating and managing campaigns, see [Qualifio Engage documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
+
+### Campaign block
+
+To promote your campaign, you can add [Campaign block](../content_management/block_reference.md#campaign-block) in Page Builder.
+In the **Campaign** tab you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
```

Comment:
```suggestion
In the **Campaign** block you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
```

Reply:
Maybe we could add here * , because by default this list contains campaigns "active or scheduled to launch in the future". But it can be changed by devs, if you configure Campaign feed in Qualifio differently. https://support.qualifio.com/hc/en-us/articles/360022954454-About-Campaign-Feeds